content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: AdoptOpenJDK 8u262, 11.0.8, and 14.0.8 Available
+date: "2020-07-16T12:00:00+00:00"
+author: andreasahlenstorf
+description: AdoptOpenJDK is happy to announce the immediate availability of AdoptOpenJDK 8u262, 11.0.8, and 14.0.2, both with OpenJDK HotSpot and Eclipse OpenJ9. As always, all binaries are thoroughly tested and available free of charge without usage restrictions on a wide range of platforms.
```

Comment:
`AdoptOpenJDK is happy to announce the immediate availability of AdoptOpenJDK 8u262, 11.0.8, and 14.0.2, both with OpenJDK HotSpot and Eclipse OpenJ9. As always, all binaries are thoroughly tested and available free of charge without usage restrictions on a wide range of platforms.`

-->

`AdoptOpenJDK is happy to announce the immediate availability of AdoptOpenJDK 8u262, 11.0.8, and 14.0.2.  Binaries are immediately available for OpenJDK and Eclipse Op[enJ9 (with OpenJDK class libraries). As always, all binaries are thoroughly tested and available free of charge without usage restrictions on a wide range of platforms.`